docs/reference/indices.asciidoc

```diff
@@ -50,12 +50,19 @@ index settings, aliases, mappings, and index templates.
 [float]
 [[index-templates]]
 === Index templates:
-* <<indices-templates>>
+
+Index templates automatically apply settings, mappings, and aliases to new indices.
+They are most often used to configure rolling indices for time series data to 
```

Comment:
It looks like we hyphenate `time series` when used as an adjective.
I don't feel strongly either way. If you decide to stick with this, I can create an issue to address the other instances.
```suggestion
They are most often used to configure rolling indices for time-series data to 
```

Reply:
The style guide says never hyphenated, so that's what I went with.